app/src/ereferrals/java/org/eyeseetea/malariacare/strategies/LoginActivityStrategy.java

```diff
@@ -74,6 +78,14 @@ public LoginActivityStrategy(LoginActivity loginActivity) {
         mPullUseCase = new PullUseCase(pullController, asyncExecutor, mainExecutor);
     }
 
+
+    private void replaceDhisLogoToHNQISLogo(LoginActivity loginActivity) {
```

Comment:
@idelcano I think that replace logos in runtime is not the better option, please you intent use only layouts, variants and resources for this issue and we should not replace in runtime resources because is not the best optime option